.github/workflows/ci.yml

```diff
@@ -63,7 +67,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install Nix
-        uses: DeterminateSystems/nix-installer-action@v2
+        uses: DeterminateSystems/nix-installer-action@v4
```

Comment:
I noticed this was the only installer action that was on v2, so I updated it as well.